### PR TITLE
R example for idr0021 rois

### PR DESCRIPTION
**Pull request by dominikl**
Simple example which reads the `batch_roi_export` table and creates and uploads a plot (which should be similar to the one in the paper).

/cc @jburel

### CODE REVIEW

**Comment by jburel** [comment]
Could you add a description to the notebook? i.e. the description of this PR

**Comment by dominikl** [comment]
Maybe I should rather call it 'Protein' and not 'Gene', didn't really want to use 'Dataset', more the biological term the dataset name stands for. Not sure about the wording in general. Maybe @will-moore can have a look at the notebook too, once it's deployed.

File: R/idr0021_rois.ipynb [hunk 1]
```diff
@@ -7,6 +7,22 @@
     "# IDR0021 - ROIs"
    ]
   },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "A simple example which reads the 'batch_roi_export' table, performs some basic statistics on it and creates a plot (which should be similar to figure 1 in the paper). The plot and results are then uploaded to server and attached to the project."
```
Comment (jburel) [inline R/idr0021_rois.ipynb]: link to the paper

**Comment by will-moore** [comment]
Yes, in general these are different proteins rather than genes.

**Comment by dominikl** [comment]
Reduced the number of steps to click through. Now there is one step for...
- Connecting to the server
- Loading the table
- Preparing the data (some reformatting, getting the image-dataset relation in, etc.)
- Plot the data
- Doing some statistics
- Upload the plot / results and disconnect

Also there now is some output for every block. I think there was an option to limit the output of a cell, do you still remember how that was done @jburel ? If I print out an R dataframe the output is limited, but it's still a bit too long. Wonder if I could limit it a bit more.  In Jupyter I can set a cell's output to be scrollable, too, but that apparently is not saved, respectively ignored.

With respect to make it more interactive: I think there wouldn't be a problem for the users to spin up the notebook and log in as "user-1" (or whatever their username will be). They can load and play around with the data themselves. Apart from the last part, which uploads the result. Well, I guess they would be able to do that, too, if it's a read-annotate group, but you'll end up with loads of attachments on the project then.

**Comment by jburel** [comment]
I will retest tomorrow since I made a final change to #6

**Comment by jburel** [comment]
@dominikl I have added entries to README in #6 It will conflict otherwise if you update it in this PR

**Comment by dominikl** [comment]
Was an error in the notebook, sorry for the hassle! Should have tried it locally earlier, would have got me a more useful error message.

**Comment by jburel** [comment]
rebuilding now

**Comment by jburel** [comment]
It is working now and the steps have been reduced.
Let's merge so @manics can deploy the image so we can test in the context of vae
We can always improve the display in a follow up PR

**Pull request merged**